admin_solutions/master_node_config.adoc

```diff
@@ -0,0 +1,409 @@
+= Master and Node Configuration
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+The master and node configuration files are the control center of the OpenShift
+cluster. This topic covers the many options available for customizing your
+OpenShift masters and nodes, and shows you how to make changes to their
+configuration after installation.
+
+The
+link:../install_config/master_node_configuration.html#master-configuration-files[*_master-config.yaml_*] and
+link:../install_config/master_node_configuration.html#node-configuration-files[*_node-config.yaml_*]
+files define a wide range of options that can be configured on the
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#master[OpenShift master] and
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#node[nodes]. These options include overriding the default plug-ins, connecting to etcd, automatically creating service accounts, building image names, customizing project requests, configuring volume plug-ins, and much more.
+
+== How Many Masters Do I Need?
+
+For the purposes of
+link:../admin_guide/high_availability.html[high availability]
+(HA), it is a good idea to use
+link:../install_config/install/advanced_install.html#multiple-masters[multiple masters].
+A cluster architecture of three masters is recommended, and
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#master[HAproxy] is the recommended solution for this.
+
+[CAUTION]
+====
+If etcd is being installed on the master hosts, then do _not_ configure your
+cluster to use two masters, because etcd will not be able to decide which one
+is authoritative when there are only two cluster members.
+If you are hosting etcd on different hosts, then it is possible to run only two
+masters.
```

Comment:
I think this might be vague. I'll make a suggestion, but I might be off course:

"Do not install etcd on a master host in an OpenShift cluster with multiple master hosts, because etcd will not be able to decide which host is authoritative. Host etcd on a separate host if your cluster has multiple master hosts."